go/service/gregor.go

```diff
@@ -146,7 +146,7 @@ func newGregorClient(g *libkb.GlobalContext) (*grclient.Client, error) {
 	g.Log.Debug("gregor handler: restoring state from leveldb")
 	if err = gcli.Restore(); err != nil {
 		// If this fails, we'll keep trying since the server can bail us out
-		g.Log.Info("gregor handler: restore local state failed: %s", err)
+		g.Log.Debug("gregor handler: restore local state failed: %s", err)
```

Comment:
If we haven't ever saved anything before, then this will trigger. That means if we do something on the client before we get a message from gregor, the user would see this get printed. Didn't seem ideal to me.